installation/nuget.md

```diff
@@ -53,7 +53,7 @@ Refer to the [Microsoft documentation about using packages in Visual Studio](htt
 
 ## Use the .NET CLI
 
-When adding NuGet sources from the .NET CLI, the credentials are stored inside the `NuGet.Config` file. The [password will be encrypted only on Windows](#store-encrypted-credentials). On other platforms, store the password in plain text or use a [NuGet API Key](#use-nuget-api-key) instead of a password.
+When adding NuGet sources from the .NET CLI, the credentials are stored in the `NuGet.Config` file. The [password can be encrypted on Windows, but with limitations](#store-encrypted-credentials). You can use a plain text password, but better [generate a NuGet API Key](#use-nuget-api-key) and use it with the .NET CLI instead of a password.
```

Comment:
```suggestion
When adding NuGet sources from the .NET CLI, the credentials are stored in the `NuGet.Config` file. The [password can be encrypted on Windows, but with limitations](#store-encrypted-credentials). You can use a plain text password, but for better security, [generate a NuGet API Key](#use-nuget-api-key), and use it with the .NET CLI instead of a password.
```